app/app.go

```diff
@@ -574,7 +591,9 @@ func NewApplication(
 
 	// The last arguments can contain custom message handlers, and custom query handlers,
 	// if we want to allow any custom callbacks
-	supportedFeatures := "iterator,staking,stargate"
```

Comment:
Reason to remove `staking` from here?

Reply:
I don't think we are querying the `staking` module. So I removed it.